docs/sphinx/_toc.yml.in

```diff
@@ -1,24 +1,34 @@
+# Anywhere {branch} is used, the branch name will be substituted.
+# These comments will also be removed.
 defaults:
   numbered: False
-  maxdepth: 6
+  maxdepth: 7
 root: index
 subtrees:
-- entries:
-  - file: releasenotes
-  - file: citation
-  - file: install
-  - file: embed
-  - file: driver
-  - file: DebugAndLogging
-  - file: cache
-  - file: perfdatabase
-  - file: finddb
-  - file: find_and_immediate
-  - file: Getting_Started_FusionAPI
-  - file: MI200AlternateImplementation
-  - file: MIOpen_Porting_Guide
-  - file: apireference
-- caption: About
+- caption: What is MIOpen?
   entries:
-    - file: license
-
+  - file: find_and_immediate.rst
+  - file: finddb.rst
+  - file: cache.rst
+  - file: perfdatabase.rst
+  - file: Getting_Started_FusionAPI.rst
+  - file: DebugAndLogging.rst
+  - file: MI200AlternateImplementation.rst
+  - file: MIOpen_Porting_Guide.rst
+  
+- caption: Quick-start
+  entries:
+  - file: reference/install.rst
+  - file: reference/embed.rst
+  - file: reference/driver.rst
+- caption: API reference
+  entries:
+  - file: reference/apireference.rst
```

Comment:
the files here in the TOC are relative to the `docs/` folder

for this to render correctly, `apireference.rst` needs to go into a folder named `reference`

Reply:
same goes for tutorials